frontend/src/app/login/login.component.spec.ts

```diff
@@ -1,20 +1,26 @@
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
 import { FormsModule } from '@angular/forms';
+import { RouterTestingModule } from '@angular/router/testing';
 
 import { AuthService } from '../auth.service';
 import { LoginComponent } from './login.component';
+import { PasswordResetComponent } from '../password-reset/password-reset.component';
```

Comment:
There's an issue on Travis with test in this module - I understand that You want here to ommit issue with `routerLink` but (at least for now) there's no reason to use fully functional `PasswordResetComponent` here.

You can create dummy `MockPasswordResetComponent`, decorate it with `Component` and decalere as component for `password-reset` path.

Something like this:
```
@Component({
  template: ''
})
class MockPasswordResetComponent {
}
```